src/Misc/layoutbin/en-US/strings.json

```diff
@@ -403,7 +403,7 @@
   "OverwriteAutoLogon": "Want to overwrite the existing autologon settings as autologon is already enabled for the user '{0}'? (Y/N)",
   "ParentTimelineNotCreated": "Parent timeline record has not been created for this new timeline record.",
   "Password": "password",
-  "PathNotExist": "Path not exist: {0}",
+  "PathDoesNotExist": "Path not exist: {0}",
```

Comment:
haha, should we also change the string here? "Path *does* not exist"?

Reply:
fixed